api/gateway/share.go

```diff
@@ -20,7 +19,7 @@ const (
 	namespacedDataEndpoint   = "/namespaced_data"
 )
 
-var nIDKey = "nid"
+var namespaceKey = "nid"
```

Comment:
I didn't change this because it's breaking gateway and CNC. I wanted to let you know that it's up for discussion.

We are not deprecating this, so it might be worth it. Does it need changes besides go-cnc?

Reply:
Let`s try not to break go-cnc before we update to opernrpc. After migration when we rely on openrpc you can break it again as we are deprecating it.